include/picongpu/particles/Particles.hpp

```diff
@@ -88,7 +91,7 @@ namespace picongpu
                       // fallback if the species has not defined the alias boundaryCondition
                       pmacc::HandleGuardRegion<
                           pmacc::particles::policies::ExchangeParticles,
-                          particles::boundary::CallPluginsAndDeleteParticles>>::type>,
+                          pmacc::particles::policies::DeleteParticles>>::type>,
```

Comment:
Discussed offline, this should become an empty functor because particle deleting will be handled by the particle boundary condition implementation.

Reply:
Fixed